ci/slab.toml

```diff
@@ -1,11 +1,9 @@
 # This profile is dedicated to test action behavior by replicating
-#profiles that can be found on repository using this workflow.
-[profile.ci-test]
+# profiles that can be found on repository using this workflow.
+[backend.aws.ci-test]
```

Comment:
you need the prefix ?

Reply:
Yes, in order to ease management on Slab side, and to avoid any confusion on user side when declaring in their own slab.toml file.